File: car-dealership-frontend/src/app/model/CreateReservation.ts
export class CreateReservation {
  vehicleId: string;
  clientId: string;
  payment: string;
  date: string;
  offeredPrice: number;
  constructor() {
    this.vehicleId = '';
    this.clientId = '';
    this.payment = '';
    this.date = '';
    this.offeredPrice = 0;
  }
}

File: CarDealership/src/main/java/rs/ac/uns/ftn/carDealership/model/dto/CreateReservation.java
package rs.ac.uns.ftn.carDealership.model.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class CreateReservation {
    String vehicleId;
    String clientId;
    String offeredPrice;
    String payment;
    String date;
}
